project_event/models/task_category_type.py

```diff
@@ -5,9 +5,12 @@
 from odoo import fields, models
 
 
-class ProjectEventType(models.Model):
-    _name = 'project.event.type'
+class TaskCategoryType(models.Model):
+    _name = 'task.category.type'
 
     name = fields.Char(
         string='Name',
     )
+    sequence = fields.Integer(
```

Comment:
@joaoalf it is not required by the client but i think that we will use the sequence order later. I have to add order by sequence now?

Reply:
@rimbendhaou You don't have to but since you already did let's leave it that way. On the other hand, sequence is not a required field so you might consider adding something like `_order = sequence, id` or `_order = sequence, name, id`.